lisp/forge-commands.el

```diff
@@ -252,26 +252,20 @@ Prefer a topic over a branch and that over a commit."
   "View the topic at point in a separate buffer."
   (interactive)
   (if-let ((topic (forge-topic-at-point)))
-      (if (forge-issue-p topic)
-          (forge-visit-issue topic)
-        (forge-visit-pullreq topic))
+      (forge-visit topic)
     (user-error "There is no topic at point")))
 
 ;;;###autoload
 (defun forge-visit-pullreq (pullreq)
   "View the pull-request at point in a separate buffer."
-  (interactive (list (forge-read-pullreq "View pull-request" t)))
-  (let ((magit-generate-buffer-name-function 'forge-topic-buffer-name))
-    (magit-mode-setup-internal #'forge-topic-mode (list pullreq) t))
-  (oset pullreq unread-p nil))
+  (interactive (list (forge-read-pullreq "View pull-request")))
```

Comment:
You dropped the `t` argument. Why?

Reply:
I hadn't meant to – it was an error introduced in the rebase. I thought I fixed them all though – I'll double-check tonight.